spec/shortcuts_spec.rb

```diff
@@ -138,10 +138,14 @@
     end
 
     it "creates a matrix of random numbers with defined seed value" do
-      m = NMatrix.random(2,:seed => 62)
+      m1 = NMatrix.random(2,:seed => 62)
+      m2 = NMatrix.random(2,:seed => 62)
+      m3 = NMatrix.random(2,:seed => 65)
+
+
+      expect(m1).to eq(m2)
+      expect(m1).not_to eq(m3)
 
-      expect(m.stype).to eq(:dense)
```

Comment:
Ah! Okay, I see that you *did* ask about this, and I missed it, @shailesh210. My apologies.